### ISSUE TITLE
Beit-3 , clarification on classification task and associated pretrained weights

### ISSUE THREAD

**Issue by William-Chittavong** [microsoft/unilm#1611]
Hi!
I'm am raising an issue that I encountered in BEiT3 for  ImageNet classification task. As mentioned in section 3.2, of the paper "Rather than appending a task layer to the vision encoder [DBK+20, BDPW22], we formulate the task as an image-to-text retrieval task. We use the category names as texts to construct image-text pairs. BEIT-3 is trained as a dual encoder to find the most relevant label for an image. "
I found that the released code use the vision encoder for ImageNet classification, while the retrieval model (with the dual encoder) is only trained on coco and flicker.
Please, could we have the pretrained weights for ImageNet classification with the dual encoder?
